libibverbs/verbs.h

```diff
@@ -502,6 +502,8 @@ enum ibv_wc_opcode {
 	IBV_WC_BIND_MW,
 	IBV_WC_LOCAL_INV,
 	IBV_WC_TSO,
+	IBV_WC_MASKED_COMP_SWAP,
```

Comment:
Any new ABI needs man page updates, and the man page updates need to also be posted to the mailing lists.

Is this IBTA specified?

Reply:
Thanks for the reminder, I will update the man page.

They're not IBTA specified, but they are discribed in RFC 7306. These opcodes allow users to pass in a mask which specifies the field boundary and then perform a bitwise logical AND operation on Add/Swap/Compare Data with the mask. That makes the ATOMIC operations more flexible.

They have been added in kernel(commit 5e80ba8ff0bd "IB/core: Add support for masked atomic operations") and supported by mlx4 and mlx5.